R/calculateConfusionMatrix.R

```diff
@@ -54,12 +59,26 @@ calculateConfusionMatrix = function(pred, relative = FALSE, sums = FALSE) {
   n.pred = length(resp)
   truth = getPredictionTruth(pred)
 
+  if (set != "both"){
+      assertClass(pred, classes = "ResamplePrediction")
+      subset.idx = (pred$data$set == set)
+
+      if (!any(subset.idx)){
```

Comment:
There should be a space between `)` and `{` (not sure why the lintr isn't catching that).

Reply:
Oh. I did that a few other times too, and he never mentioned it. Maybe he doesn't have a rule for it?